frontend/src/app/rest/page.ts

```diff
@@ -0,0 +1,29 @@
+
+export interface PageSort {
+  unsorted: boolean
+  sorted: boolean
+  empty: boolean
+}
+
+export interface Pageable {
+  sort: PageSort
+  offset: number
+  pageNumber: number
+  pageSize: number
+  paged: boolean
+  unpaged: boolean
+}
+
+export interface Page<T> {
+  content: Array<T>
+  empty: boolean
```

Comment:
Can't this be calculated by checking if `content` is empty? Having this as a separate property may, theoretically, lead to this being true/false while the other is not empty/empty.

Reply:
This interface mirrors Spring's `Page` interface.